Backoffice/EventSubscriber/ContentUpdateCacheSubscriber.php

```diff
@@ -64,14 +61,33 @@ public function invalidateCacheOnDeletePublishedContent(ContentEvent $event)
         }
     }
 
+   /**
+     * @param ContentEvent $event
+     */
+    public function invalidateCacheOnUpdateOutOfWorkflowContent(ContentEvent $event)
+    {
+        $content = $event->getContent();
+        $previousStatus = $event->getPreviousStatus();
+
+        if ($previousStatus instanceof StatusInterface && $previousStatus->isOutOfWorkflow()) {
```

Comment:
@itkg-nanne : should the criteria be $previousStatus->isOutOfWorkflow() or $previousStatus->isPublished() ?